test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -296,7 +296,8 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved != nil {
 				return
 			}
-			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil && !*profile.Spec.WorkloadHints.RealTime {
+			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
+				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
```

Comment:
Didn't you mean ```IsPyhsicalRpsEnabled``` instead of ```IsRpsEnabled``` ?

Reply:
no. this is the right annotation check here.